tests/test_cxl_image_classification_host.py

```diff
@@ -0,0 +1,156 @@
+    )
+
+    switch_configs = [
+        VirtualSwitchConfig(upstream_port_index=0, vppb_counts=4, initial_bounds=[1, 2, 3, 4])
+    ]
+    virtual_switch_manager = VirtualSwitchManager(
+        switch_configs=switch_configs, physical_port_manager=physical_port_manager
+    )
+
+    dev1 = MyType1Accelerator(port_index=1, port=switch_port, irq_listen_port=9100)
+    dev2 = MyType1Accelerator(port_index=2, port=switch_port, irq_listen_port=9101)
+    dev3 = MyType1Accelerator(port_index=3, port=switch_port, irq_listen_port=9102)
+    dev4 = MyType1Accelerator(port_index=4, port=switch_port, irq_listen_port=9103)
+
+    host_manager = CxlHostManager(host_port=host_port, util_port=util_port)
+    host_mem_size = 0x8000  # Needs to be big enough to test cache eviction
+
+    host_name = "foo"
+    root_port_switch_type = ROOT_PORT_SWITCH_TYPE.PASS_THROUGH
+    memory_controller = RootComplexMemoryControllerConfig(host_mem_size, "foo.bin")
+    root_ports = [RootPortClientConfig(0, "localhost", switch_port)]
+    memory_ranges = [MemoryRange(MEMORY_RANGE_TYPE.DRAM, 0x0, host_mem_size)]
+
+    config = CxlImageClassificationHostConfig(
+        host_name,
+        0,
+        root_port_switch_type,
+        memory_controller,
+        memory_ranges,
+        root_ports,
+        coh_type=COH_POLICY_TYPE.DotCache,
+    )
+
+    host_manager = CxlHostManager(host_port=host_port, util_port=util_port)
+    host = CxlComplexHost(config)
+
+    pci_bus_driver = PciBusDriver(host.get_root_complex())
+    cxl_bus_driver = CxlBusDriver(pci_bus_driver, host.get_root_complex())
+    cxl_mem_driver = CxlMemDriver(cxl_bus_driver, host.get_root_complex())
+
+    start_tasks = [
+        asyncio.create_task(host.run()),
+        asyncio.create_task(host_manager.run()),
+        asyncio.create_task(sw_conn_manager.run()),
+        asyncio.create_task(physical_port_manager.run()),
+        asyncio.create_task(virtual_switch_manager.run()),
+        asyncio.create_task(dev1.run()),
+        asyncio.create_task(dev2.run()),
+        asyncio.create_task(dev3.run()),
+        asyncio.create_task(dev4.run()),
+    ]
+
+    wait_tasks = [
+        asyncio.create_task(sw_conn_manager.wait_for_ready()),
+        asyncio.create_task(physical_port_manager.wait_for_ready()),
+        asyncio.create_task(virtual_switch_manager.wait_for_ready()),
+        asyncio.create_task(host_manager.wait_for_ready()),
+        asyncio.create_task(host.wait_for_ready()),
+        asyncio.create_task(dev1.wait_for_ready()),
+        asyncio.create_task(dev2.wait_for_ready()),
+        asyncio.create_task(dev3.wait_for_ready()),
+        asyncio.create_task(dev4.wait_for_ready()),
+    ]
+    await asyncio.gather(*wait_tasks)
+
+    async def test_configs():
+        await pci_bus_driver.init()
+        await cxl_bus_driver.init()
+        await cxl_mem_driver.init()
+
+        logger.debug(cxl_mem_driver.get_devices())
```

Comment:
Need more info than just printing devices.

Reply:
Removed for now and maybe add more debugging messages in the future when needed.